OpenRA.Game/Network/ReplayRecorderConnection.cs

```diff
@@ -79,21 +79,23 @@ void StartSavingReplay(byte[] initialContent)
 
 		public void Receive(Action<int, byte[]> packetFn)
 		{
+			if (disposed) // if has been disposed do no continue as writer nolonger exist
```

Comment:
This comment will need to be fixed.
It should go above the `if` line and grammatically corrected, if we are going to keep it.